server/internal/email_sender.go

```diff
@@ -40,7 +41,7 @@ var (
 )
 
 // SendMail sends verification mails
-func SendMail(sender, sendGridKey, receiver, subject, body string) error {
+func SendMail(sender, sendGridKey, receiver, subject, body, attachmentName string, attachmentBytes ...[]byte) error {
```

Comment:
I think we can make it a Mailer that can be initialized with a Key e.g NewMailer(sendGridKey) 

And then We can SendMail(sender, receiver, subject, body, attachments []Attachment) 

and Attachments to be a struct of a name and bytes